scripts/raceday_verify.py

```
@@ -418,6 +431,63 @@ def check_odds_coverage(con, year, monthday, issues):
+        ("TS_O1", "単複枠"),
+        ("TS_O2", "馬連"),
+        ("TS_O3", "ワイド"),
+        ("TS_O4", "馬単"),
+        ("TS_O5", "3連複"),
+        ("TS_O6", "3連単"),
+    ]
+
+    any_data = False
+    latest_hasso = None
+    for tbl, name in ts_tables:
+        cnt = q(con, f"SELECT COUNT(*) FROM {tbl} WHERE Year=? AND MonthDay=?", (y, m))
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)